iota-client/response.rs

```diff
@@ -2,6 +2,59 @@
 use anyhow::Result;
 use iota_bundle_preview::{Address, Hash, Tag, Transaction, TransactionField};
 use iota_ternary_preview::TryteBuf;
+use serde::ser::{Serialize, Serializer, SerializeSeq, SerializeStruct};
+
+#[derive(Serialize)]
+pub struct TransactionDef {
```

Comment:
I enforce a lint to write documentation. This one is needed for example.

Reply:
yeah I saw that but forgot to add it.